src/main/java/frc/team1816/robot/Robot.java

```diff
@@ -1,10 +1,39 @@
 package frc.team1816.robot;
 
-import edu.wpi.first.wpilibj.IterativeRobot;
+import com.edinarobotics.utils.gamepad.Gamepad;
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.command.Scheduler;
+import frc.team1816.robot.commands.GamepadArmCommand;
+import frc.team1816.robot.commands.GamepadDriveCommand;
+import frc.team1816.robot.commands.GamepadIntakeCommand;
+import frc.team1816.robot.commands.GamepadShooterCommand;
+import frc.team1816.robot.subsystems.Arm;
+import frc.team1816.robot.subsystems.Drivetrain;
+import frc.team1816.robot.subsystems.Intake;
+import frc.team1816.robot.subsystems.Shooter;
+
+public class Robot extends TimedRobot {
+
+    private Drivetrain drivetrain;
+    private Arm arm;
+    private Intake intake;
+    private Shooter shooter;
+
+    private Gamepad gamepadDriver, gamepadOverride;
 
-public class Robot extends IterativeRobot {
     @Override
-    public void robotInit() { }
+    public void robotInit() {
+        Components.getInstance();
+        Controls.getInstance();
+
+        drivetrain = Components.getInstance().drivetrain;
+        arm = Components.getInstance().arm;
+        intake = Components.getInstance().intake;
+        shooter = Components.getInstance().shooter;
+
+        gamepadDriver = Controls.getInstance().gamepadDriver;
+        gamepadOverride = Controls.getInstance().gamepadOverride;
```

Comment:
It doesn't look like gamepadOverride is setup, so you can probably remove this line.